doc/storage.md

```diff
@@ -10,6 +10,10 @@ Key                             | Type      | Condition
 size                            | string    | appropriate driver and source     | 0                          | Size of the storage pool in bytes (suffixes supported). (Currently valid for loop based pools and zfs.)
 source                          | string    | -                                 | -                          | Path to block device or loop file or filesystem entry
 btrfs.mount\_options            | string    | btrfs driver                      | user\_subvol\_rm\_allowed  | Mount options for block devices
+ceph.cluster\_name              | string    | ceph driver                       | ceph                       | Name of the ceph cluster in which to create new storage pools.
+ceph.osd.pool\_name             | string    | ceph driver                       | name of the pool           | Name of the osd storage pool.
+ceph.osd.pg\_num                | string    | ceph driver                       | 32                         | Number of placement groups for the osd storage pool.
+ceph.rbd.clone\_copy            | string    | ceph driver                       | true                       | Whether to use RBD lightweight clones rather than full dataset copies.
```

Comment:
I suspect that this will include more than just the ceph keys as I believe we've introduced a number of LVM and btrfs keys without updating that part of the documentation to match.